Cargo.lock

```diff
@@ -9780,9 +9749,9 @@ dependencies = [
 
 [[package]]
 name = "zeroize"
-version = "1.7.0"
+version = "1.8.1"
 source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "525b4ec142c6b68a2d10f01f7bbf6755599ca3f81ea53b8431b7dd348f5fdb2d"
```

Comment:
@jost I think this broke the build

Reply:
Okay, I'll downgrade it back. I couldn't even run this once, because of incompatible deps pulled in by hdi/hdk from deepkey_dna_sdk.